File: Google Kickstart/BikeTour/BikeTour.cpp
// Problem
// https://codingcompetitions.withgoogle.com/kickstart/round/000000000019ffc8/00000000002d82e6

#include <bits/stdc++.h>
using namespace std;

int main()
{
    int T, N, H, temp, count = 0;
    vector<int> peaks;
    cin >> T;

    for (int i = 0; i < T; i++)
    {
        cin >> N;
        for (int j = 0; j < N; j++)
        {
            cin >> temp;
            peaks.push_back(temp);
        }

        for (int k = 1; k < peaks.size() - 1; k++)
        {
            if (peaks[k] > peaks[k - 1] && peaks[k] > peaks[k + 1])
                count++;
        }
        peaks.clear();
        cout << "Case #" << i + 1 << ": " << count << "\n";
        count = 0;
    }

    return 0;
}

File: Google Kickstart/BikeTour/BikeTour.py
# Problem
# https://codingcompetitions.withgoogle.com/kickstart/round/000000000019ffc8/00000000002d82e6

T, count = int(input()), 0

for _ in range(T):
    N = int(input())
    peaks = [int(i) for i in input().split(' ')]
    # 10 3 10
    for j in range(1, len(peaks)-1):
        if peaks[j] > peaks[j-1] and peaks[j] > peaks[j+1]:
            count += 1
    print(f'Case #{_+1}: {count}')
    count = 0
    peaks.clear()
